Description: Fix secret key path
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-09-28

--- ironic-ui-2.0.0.orig/ironic_ui/test/settings.py
+++ ironic-ui-2.0.0/ironic_ui/test/settings.py
@@ -38,3 +38,6 @@ settings.update_dashboards(
 
 # Ensure any duplicate apps are removed after the update_dashboards call
 INSTALLED_APPS = list(set(INSTALLED_APPS))
+
+SECRET_KEY = secret_key.generate_or_read_from_file(
+    os.path.join(__file__, '.secret_key_store'))
